src/runtime/node/internal/process/process.ts

```diff
@@ -92,11 +94,11 @@ export class Process extends EventEmitter implements NodeJS.Process {
   ppid: number = 100;
 
   get version() {
-    return "";
+    return `v${NODE_VERSION}`;
```

Comment:
I note that one can attempt to write to `process.version` in Node.js it just doesn't update:

```
> process.version
'v18.20.6'
> process.version = "xxx"
'xxx'
> process.version
'v18.20.6'
```

with this just being a getter is that going to break with `process.version = "xxx"`?

Reply:
This is same behavior of class getters

```js
// 1
c=new (class { get version(){return "1"} })();c.version=2;console.log(c.version) 
```